src/ai/defaultai.h

```diff
@@ -392,10 +395,11 @@ struct DefaultAI : ComputerPlayer {
 	   "trident", "tabard", "shield", "mask",    "spear", "warrior"};
 
 	// seafaring related
-	enum { kReprioritize, kStopShipyard, kStapShipyard };
+	enum { kReprioritize, kStopShipyard, kStartShipyard };
 	static Time last_seafaring_check_;
 	// False by default, until Map::allows_seafaring() is true
 	static bool map_allows_seafaring_;
+	bool potential_wrong_shipyard_ = false;
```

Comment:
needed to cope for the (now more rare) case that a shipyard is built at the wrong sea